ledger/ledger-utxo/scans/corda-ledger-utxo-5.2.0.yaml

```diff
@@ -421,6 +421,17 @@ net.corda.v5.ledger.utxo.UtxoLedgerService:
           annotation:
             - NotNull
           type: net.corda.v5.ledger.utxo.transaction.UtxoTransactionValidator
+    receiveTransaction:
```

Comment:
@blsemo do you have any information on what this file is for because this is new to me?

Reply:
Is not that related to this PR? https://github.com/corda/corda-api/pull/1274

Reply:
this is to be able to raise breaking change PR, it looks like we now must run `./gradlew cementApi` to raise it.

Reply:
What does it define as breaking changes? 🤔 We didn't break anything...

Reply:
adding new APIs is breaking change it looks like.